airbyte-integrations/connectors/source-vitally/source_vitally/manifest.yaml

```diff
@@ -1,105 +1,895 @@
+            page_size: 100
+            cursor_value: "{{ response.next }}"
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/organizations"
+    tasks:
+      type: DeclarativeStream
+      name: tasks
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /tasks
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - results
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            inject_into: request_parameter
+            field_name: from
+          page_size_option:
+            type: RequestOption
+            field_name: limit
+            inject_into: request_parameter
+          pagination_strategy:
+            type: CursorPagination
+            page_size: 100
+            cursor_value: "{{ response.next }}"
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/tasks"
+    users:
+      type: DeclarativeStream
+      name: users
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /users
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - results
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            inject_into: request_parameter
+            field_name: from
+          page_size_option:
+            type: RequestOption
+            field_name: limit
+            inject_into: request_parameter
+          pagination_strategy:
+            type: CursorPagination
+            page_size: 100
+            cursor_value: "{{ response.next }}"
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/users"
+  base_requester:
+    type: HttpRequester
+    url_base: https://rest.vitally.io/resources/
     authenticator:
       type: BasicHttpAuthenticator
       username: "{{ config['api_key'] }}"
-  retriever:
-    record_selector:
-      $ref: "#/definitions/selector"
-    paginator:
-      type: DefaultPaginator
-      pagination_strategy:
-        type: "CursorPagination"
-        cursor_value: "{{ response.next }}"
-        page_size: 100
-      page_size_option:
-        field_name: "limit"
-        inject_into: "request_parameter"
-      page_token_option:
-        type: RequestOption
-        field_name: "from"
-        inject_into: "request_parameter"
-    requester:
-      $ref: "#/definitions/requester"
+      password: "{{ config['nothing'] }}"
```

Comment:
can we set this directly to an empty string and remove the `nothing` param from the config?

Reply:
Currently the builder does not supports empty password field, so that was a workaround of having a non existent config parameter

Reply:
@btkcodedev I'd expect the builder to fallback to YAML-component if the password field is empty. Is that not how it behaves?

Reply:
not actually, it throws error like password should be in form of {{config['password']}}